### PR TITLE
Features/markdown

### PR DESCRIPTION
**Pull request by maximihajlov**
## New features to test

Adds markdown support, new description field and AI-powered function to automatically generate descriptions. 

### Markdown support:
Markdown support example

### Code blocks
Code block example

### AI generated descriptions

https://user-images.githubusercontent.com/43277901/233787661-afe3bb4f-ddd3-4826-b281-0edd3b3e293a.mp4

### New category selection 

image

### CODE REVIEW

File: backend/database/migrate/migrations/v003.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import "database/sql"
+
+// v003 adds a description field to the post table
+//
+// For already existing posts, the description is set to the first 200 characters of the content
```
[the remark this answers is not in the document]
Comment (maximihajlov) [reply backend/database/migrate/migrations/v003.go:7]: fixed in 5327fca

**Review by nattikim — CHANGES_REQUESTED** [review]
Content text is not cropping

**Comment by maximihajlov** [comment]
Fixed in 5327fca

**Review by nattikim — APPROVED** [review]

**Review by nattikim — APPROVED** [review]
LGFM. But couldn't test the description generator (don't have the Key) 🔑

**Pull request merged**